src/ext/java/org/opentripplanner/ext/fares/impl/DefaultFareService.java

```diff
@@ -33,7 +33,7 @@
 class FareSearch {
 
   // Cell [i,j] holds the best (lowest) cost for a trip from rides[i] to rides[j]
-  Money[][] resultTable;
+  float[][] resultTable;
```

Comment:
This is a private field, right?

Reply:
The whole class is package-private.

I really tried understanding the algorithm and simplifying it but I failed.